extensions/sabnzbd/src/change-complete-action.tsx

```diff
@@ -1,36 +1,17 @@
-import { showToast, ToastStyle, preferences, Form, ActionPanel, Icon, SubmitFormAction } from "@raycast/api";
-import { Client, Results, CompleteAction } from "sabnzbd-api";
+import { LaunchProps, showToast, Toast } from "@raycast/api";
+import { client } from "./sabnzbd";
 
-export default function Command() {
-  return (
-    <Form
-      actions={
-        <ActionPanel>
-          <ChangeCompleteActionAction />
-        </ActionPanel>
-      }
-    >
-      <Form.Dropdown id="action" title="Action" storeValue defaultValue={CompleteAction.ShutdownProgram}>
-        <Form.Dropdown.Item value={CompleteAction.ShutdownProgram} title="Shutdown Program" />
-        <Form.Dropdown.Item value={CompleteAction.HibernatePC} title="Hibernate PC" />
-        <Form.Dropdown.Item value={CompleteAction.StandbyPC} title="Standby PC" />
-      </Form.Dropdown>
-    </Form>
-  );
-}
-
-function ChangeCompleteActionAction() {
-  async function handleSubmit(values: any) {
-    const client = new Client(preferences.url.value as string, preferences.apiToken.value as string);
-
-    try {
-      const results = (await client.changeCompleteAction(values.action)) as Results;
-      showToast(ToastStyle.Success, "Changed complete action");
-    } catch (error) {
-      console.error(error);
-      showToast(ToastStyle.Failure, "Could not change complete action");
-    }
+export default async (props: LaunchProps<{ arguments: Arguments.ChangeCompleteAction }>) => {
+  const { action } = props.arguments;
+  const toast = await showToast({ style: Toast.Style.Animated, title: "Changing Complete Action" });
+  try {
+    await client.changeCompleteAction(action).then((result) => {
+      if (!result.status || result.error) throw new Error();
+    });
+    toast.style = Toast.Style.Success;
+    toast.title = "Changed Complete Action";
+  } catch (error) {
+    toast.style = Toast.Style.Failure;
+    toast.title = "Could not Change Complete Action";
```

Comment:
**suggestion:** My personal preference is to use sentence case throughout, aka lowercase for "Change Complete Action". There's no styleguide though for Sabnzbd capitalisation though ;)

Reply:
We can sentence case it! But wouldn't it make sense to treat "Complete Action" as a proper noun i.e. "Could not change Complete Action" rather than "Could not change complete action"?
Will update the other toasts.

Reply:
I can work through in my mind, in which both ways seem correct, haha. I'm not fussed, so feel free to go with what you think is best.